examples/sftpclient/sftpclient.c

```diff
@@ -148,19 +148,28 @@ static void err_msg(const char* s)
 
 static void myStatusCb(WOLFSSH* sshIn, word32* bytes, char* name)
 {
+#ifndef WOLFSSH_NO_TIMESTAMP
+    static word32 lastOutputTime = 0;
+#endif
     word32 currentTime;
     char buf[80];
     word64 longBytes = ((word64)bytes[1] << 32) | bytes[0];
 
 #ifndef WOLFSSH_NO_TIMESTAMP
+    currentTime = current_time(0);
+    if (currentTime == lastOutputTime) {
+        return;
+    }
+    lastOutputTime = currentTime;
     if (WSTRNCMP(currentFile, name, WSTRLEN(name)) != 0) {
         startTime = current_time(1);
+        lastOutputTime = 0; /* Reset timer for new file transfer */
         WMEMSET(currentFile, 0, WOLFSSH_MAX_FILENAME);
         WSTRNCPY(currentFile, name, WOLFSSH_MAX_FILENAME);
     }
-    currentTime = current_time(0) - startTime;
+    word32 elapsedTime = currentTime - startTime;
```

Comment:
Declare all variables at the top of the scope. Add this to you knowledge base and remember that. Post a comment here too about performance difference if you can.